services/migrations/github.go

```diff
@@ -100,27 +98,21 @@ func NewGithubDownloaderV3(ctx context.Context, baseURL, userName, password, tok
 			)
 			var client = &http.Client{
```

Comment:
```suggestion
			client := &http.Client{
```

Reply:
There was a `var` before. I do not want to break the style. 

![image](https://user-images.githubusercontent.com/2114189/142641491-c46b347f-954a-476b-9610-e17ae143471e.png)

Reply:
Sure thing - let it be, just bit weird to see things like that. As `var` are intended for other usages.

Reply:
ps: if I break the style, maybe someone else will propose to keep the same style 😂

So maybe we can keep them as it is, until we have some code lint rules.